incoming/item_events.md

```diff
@@ -46,41 +46,41 @@ The data model depends on the event type, see below.
-| `freightGroup`              | string  | **Optional** |
-| `hasDurabilityDate`         | boolean | **Required** |
-| `hazardLabels`              | array of string   | **Optional** |
-| `launchDate`                | string  | **Optional** | yyyy-MM-dd
-| `manufacturerItemNumber`    | string  | **Required** |
-| `marketingText`             | string  | **Optional** |
-| `modelName`                 | string  | **Optional** |
-| `moduleNumber`              | integer | **Required** |
-| `nrfInfo`                   | object  | **Optional** |
-| `priceUnit`                 | string  | **Required** |
-| `primaryText`               | string  | **Required** | "Varetekst 1"
-| `productNumber`             | integer | **Optional** | Reference to Product
-| `itemOwnerItemNumber`       | string  | **Required** | 
-| `replacesNobbNumber`        | integer | **Optional** |
-| `secondaryText`             | string  | **Optional** | "Varetekst 2" in NOBB domain language.
-| `seriesName`                | string  | **Optional** |
-| `stocked`                   | boolean | **Required** |
-| `tax`                       | string  | **Optional** |
-| `toleratesFrost`            | boolean | **Optional** |
-| `tunNumber`                 | string  | **Optional** |
-| `type`                      | string  | **Required** | Type of item. One of "Standard, Display, Composite, Service"
-| `uniqueSellingPoints`       | array of string   | **Optional** |
+| Property                    | Type    | Required     | Description  | Riversand Comment
+| --------------------------- | ------- | ------------ | ------------ | ---------------------------------------------------------------------------------
+| `id`                        | string  | **Required** | GUID (must be generated and can't be changed) | This will be generated and stored in Middleware.
+| `nobbNumber`                | integer | **Required** | Must be generated and can't be changed. 8 digits. | thgnobbno
+| `accessories`               | array of integers | **Optional** | Array of NOBB numbers. | TBD
+| `bundleItems`               | array of objects  | **Optional** | Only used for Display and Composite items | TBD 
+| `customsEuCode`             | string  | **Optional** | | TBD
+| `customsNoCode`             | string  | **Optional** | | TBD
+| `dangerousGoods`            | object  | **Optional** | | thgpackaginglabels
+| `description`               | string  | **Optional** | | thgdescription
+| `environmentLabels`         | array of string   | **Optional** | | thgenvironmentlabels
+| `expiryDate`                | string  | **Optional** | yyyy-MM-dd | thgexpiredate
+| `finfoNumber`               | string  | **Optional** | | thgfinfono
+| `freightGroup`              | string  | **Optional** | | thgfreightgroup
+| `hasDurabilityDate`         | boolean | **Required** | | thghasdurabilitydate
+| `hazardLabels`              | array of string   | **Optional** | | thgdangerclass
```

Comment:
This has values like "Poisonous", "Explosive" and so on. It is also called packaging labels.
```suggestion
| `hazardLabels`              | array of string   | **Optional** | | thgpackaginglabels
```

Reply:
Changed 'hazardLabels' mapped to thgpackaginglabels, instead of mapped to thgdangerclass